Formula/wrk.rb

```diff
@@ -14,21 +22,21 @@ class Wrk < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "971a44b8fc296eb7148763277306af8e022210f4ed518a0a290c82730bd1bfef"
   end
 
-  depends_on "openssl@1.1"
-
-  uses_from_macos "unzip" => :build
-
-  on_linux do
-    depends_on "makedepend" => :build
-    depends_on "pkg-config" => :build
-  end
+  depends_on "luajit"
+  depends_on "openssl@3"
 
   conflicts_with "wrk-trello", because: "both install `wrk` binaries"
 
   def install
     ENV.deparallelize
     ENV["MACOSX_DEPLOYMENT_TARGET"] = MacOS.version
-    system "make"
+    ENV.append_to_cflags "-I#{Formula["luajit"].opt_include}/luajit-2.1"
```

Comment:
This seems like it should be fixed upstream

Reply:
https://github.com/wg/wrk/pull/320 would probably fix it, but PR has been open since 2018 with no comments from upstream.